### DISCUSSION TITLE
I Hope Entry Multi Styling is Available in MAUI.

### DISCUSSION THREAD

**Discussion by nextcodelab** [dotnet/maui#2164] [General]
I now facing the biggest problems in xamarin forms including buggy modal navigation page...
I want to style entry like just simple removing underline but is is not available out of the box.... The styling available in are just maybe 3 properties Fontsize, FontAtributes, TextColor and Fontsize... This is very limited styling! I hope I am a real graphics programmer so I can create my own controls from graphics class and add hundreds of styling properties to entry with unlimited design capabilities.

I still try to hide underline by adding Grid as Cover to entry underline,, and it works. But another problems occurred, the hot reload only works on first Page but in Modal Pages hot reload is not working... And all exemption are not break to specific error instead go to non framework thread warning... 

Since it seems xamarin buggy I will try to try implementing Await Task Run at all cost to make my app perform well... The CarouselView is super Buggy And Selected currentItem does not changed its positions sometimes. and there are no event  in carouselview that items is loaded so that we can assign item position.

I hope this will change in .net MAUI in the future.

**Comment by GalaxiaGuy** [comment]
Xamarin Community Toolkit has an effect for removing the border on Entry:

https://github.com/xamarin/XamarinCommunityToolkit/tree/main/src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/RemoveBorder
https://github.com/xamarin/XamarinCommunityToolkit/blob/main/samples/XCT.Sample/Pages/Effects/RemoveBorderEffectPage.xaml

Since Xamarin Community Toolkit became "blessed" by Microsoft, I believe that (and now Maui Community Toolkit) is the place where less "core" functionality will be developed.

The new handler system does make platform specific behavior much easier to implement.

**Reply by nextcodelab** [reply]
I found this library [https://github.com/dotnet/Microsoft.Maui.Graphics.Controls](https://github.com/dotnet/Microsoft.Maui.Graphics.Controls)
